### ISSUE TITLE
Powershell Core on MacOS erroneously reports invalid characters

### ISSUE THREAD

**Issue by andyhopp** [EliteLoser/WriteAscii#6]
After installing the module on MacOS with Powershell Core, the Write-Ascii cmdlet reports errors:
```
/Users/andyhopp> Write-Ascii "Hello, World!"
sort: No such file or directory
Unsupported character, using these accepted characters: .
```

**Comment by EliteLoser** [comment]
In this old script I used `Sort` instead of `Sort-Object`, and this causes errors on Linux and MacOS. Thanks for fixing it in your pull request. It's merged now. I have to figure out a way to update the PowerShell Gallery...

**Comment by EliteLoser** [comment]
(Fixed in #7 )
